### PR TITLE
BYT-710: Updated store-locator version and exposed more options

### PR DESCRIPTION
**Pull request by jgettings**
*Note: yarn.lock will need to be updated once the new version of `store-locator` is live

We needed access to the `StoreLocatorMap` type and it was not exposed yet.

The `storeListOptions` should've been in here, but it was missed when that was added.

### CODE REVIEW

File: example/index.tsx
Comment (dmccabe) [inline example/index.tsx:33]: Should these `console.log` statements be here?

Comment (jgettings) [reply example/index.tsx:33]: This is the example site that you run locally for developing, so I left it in there intentionally for testing. I should probably add the new `storeList` object though.

Comment (jgettings) [reply example/index.tsx]: @dmccabe this PR is still waiting your approval - I can remove the console.logs if you prefer 😂

Comment (dmccabe) [reply example/index.tsx]: Hah nope were good, was just waiting for a response before approval, I missed the notification.

**Review by dmccabe — APPROVED** [review]

**Pull request merged**
